.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: Set up JDK 11
```

Comment:
Maybe nice to update the version in name also.

Reply:
Done, thanks.